Documentation/usage/dlv_substitute-path-guess-helper.md

```diff
@@ -0,0 +1,39 @@
+## dlv substitute-path-guess-helper
+
+
```

Comment:
since this is a public page :-) I think it's worth mention that this is an internal utility to support DAP's  remote debugging guess substitute path.

Reply:
Actually, I don't think that page should exist at all, since it's an hidden command. I've removed it. Thanks.